package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Vitor Buzinaro <buzinas@buzinas.com>",
   "license": "MIT",
   "devDependencies": {
+    "any-promise": "^1.3.0",
```

Comment:
Please, change `any-promise` to `es6-promise` (it already comes with `.d.ts` and you can use ES6 imports.